website/snippets/grants-vs-access-to.md

```diff
@@ -1,8 +1,8 @@
 :::info Note
-The `grants` config is unrelated to the `grant_access_to` config:
+The [`grants` config](grants) and the [`grant_access_to` config](bigquery-configs#authorized-views) are distinct.
 
 - **`grants_access_to`:** Enables you to set up authorized views. When configured, dbt provides an authorized view access to show partial information from other datasets, without providing end users with full access to those underlying datasets. For more information, see [BigQuery configurations: Authorized views](/reference/resource-configs/bigquery-configs#authorized-views)
 - **`grants`:** Provides specific permissions to users, groups, or service accounts for managing access to datasets you're producing with dbt.
 
-You can use the two features together: "authorize" a view model with the `grants_access_to` configuration, and then add `grants` to that view model to share its query results (and _only_ its query results) with other users, groups, or service accounts. For more information, see [grants](/reference/resource-configs/grants).
```

Comment:
I had added this because the snippet is also in https://deploy-preview-1707--docs-getdbt-com.netlify.app/reference/resource-configs/bigquery-configs#authorized-views. But I'm fine with cutting it if you think it's confusing.

Reply:
Totally makes sense to have the snippet in both places! I only removed the link here because I added the links to both pages above — that way, it made more sense to me in both locations